hilda/snippets/dyld.py

```diff
@@ -1,39 +1,126 @@
-import logging
+from functools import cached_property
```

Comment:
i wonder if theres a chance we'll need to support python3.7 or lower? since `cached_property` is only available from python3.8+. i think a better choice would be adding dependency on `cached-property` into requirements and change to:

```python
from cached_property import cached_property
```

Reply:
forgot to add into module's requirements (`setup.cfg)